docs/src/app/components/pages/components/Subheader/Page.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import CodeExample from '../../../CodeExample';
+import PropTypeDescription from '../../../PropTypeDescription';
+import MarkdownElement from '../../../MarkdownElement';
+
+import subheaderReadmeText from './README';
+import listExampleChatCode from '!raw!../List/ExampleChat.jsx';
+import ListExampleChat from '../List/ExampleChat.jsx';
+import listExampleFoldersCode from '!raw!../List/ExampleFolders.jsx';
+import ListExampleFolders from '../List/ExampleFolders.jsx';
+import gridListExampleSimpleCode from '!raw!../GridList/ExampleSimple';
+import GridListExampleSimple from '../GridList/ExampleSimple';
+import subheaderCode from '!raw!material-ui/lib/Subheader/Subheader';
+
+const SubheaderPage = () => (
+  <div>
+    <MarkdownElement text={subheaderReadmeText} />
+    <CodeExample
+      title="Simple List with Subheader"
```

Comment:
No, you're English was fine! It's really just a small adjustment in terms of which words are "highlighted" first in the title.

@mbrookes Does this change make sense?